lib/src/types/c2_detailed_result.dart

```diff
@@ -0,0 +1,11 @@
+part of 'index.dart';
+
+class C2DetailedResult {
```

Comment:
Is there a reason this class exists as a separate thing from `C2WebhookResult`? it seems to borrow similar comments from that class and serve the same intended function.

Reply:
the webhook almost gives you all the data you need (except the number of intervals) and making a separate REST call (which may not be fully propagated across C2's network - fun fact) to the c2logbook API  to retrieve this extra data